File: examples/eApi v1.9/java/eAPI-v19-connector/src/main/java/cz/monetplus/mips/eapi/v19/connector/entity/PaymentCloseRequest.java
package cz.monetplus.mips.eapi.v19.connector.entity;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.*;

@EqualsAndHashCode(callSuper = false)
@RequiredArgsConstructor
@AllArgsConstructor
@Data
@JsonInclude(value = JsonInclude.Include.NON_NULL)
public class PaymentCloseRequest  extends SignBase {
    @NonNull
    private  String merchantId;
    @NonNull
    private String payId;
    private Long totalAmount;

    @Override
    public String toSign() {
        StringBuilder sb = new StringBuilder();
        add(sb, merchantId);
        add(sb, payId);
        add(sb, dttm);
        add(sb, totalAmount);
        deleteLast(sb);
        return sb.toString();
    }
}

File: examples/eApi v1.9/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/PaymentCloseRequest.cs
using System.Text;
using CsobGatewayClientExample.Communication.DataObjects;
using Newtonsoft.Json;

namespace CsobGatewayClientExample.Communication;

public class PaymentCloseRequest : SignBaseRequest
{
    [JsonProperty("payId")]
    public string? PayId;
    [JsonProperty("totalAmount")]
    public long? TotalAmount;

    public override string ToSign()
    {
        StringBuilder sb = new StringBuilder();
        Add(sb, MerchantId);
        Add(sb, PayId);
        Add(sb, Dttm);
        Add(sb, TotalAmount);
        DeleteLast(sb);
        return sb.ToString();
    }
}